src/components/SnapshotSource.vue

```diff
@@ -3,12 +3,14 @@
         <div class="SnapshotSource-content has-background-grey-lighter">
             <iframe 
                 ref="source"
-                id="source" 
-                :src="buildSnapshotUrl(snapshotId)" 
+                id="source"  
+                :src="snapshotUrl"
+                :key="'codepress-iframe-'+snapshotUrl"
                 :width="viewportSize.width" 
                 frameborder="0"
                 @load="onIframeLoaded"
                 v-show="loaded"
+                sandbox="allow-scripts allow-same-origin"
```

Comment:
I would not allow scripts. The page should look as close to the original as possible, but should not be "usable". Besides script tags are removed anyway from the snapshot.

Reply:
if I remove allow-scripts from the sandbox attribute then I get errors like 

> Blocked autofocusing on a form control because the form's frame is sandboxed and the 'allow-scripts' permission is not set.

should i remove the sandbox attribute?

Reply:
OK, Keep it like this